### PR TITLE
OY2-16868: Add Jira Issue Collector Feedback Tab to Dev Environment

### PR DESCRIPTION
**Pull request by haworku**
## Summary
Add Jira Issue Collector Feedback Tab to Dev Environment

Notes: 
- added a useScript hook to handle this after trying a couple approaches that didn't work
- More about the ticket collector: https://support.atlassian.com/jira-cloud-administration/docs/use-the-issue-collector/
- Also saw this: _The JavaScript exposed by the issue collector is not considered a stable API and may change with new Jira releases._

### Screenshots
Screen Shot 2022-03-29 at 4 06 22 PM

#### Related issues
https://qmacbis.atlassian.net/browse/OY2-16868

### CODE REVIEW

**Comment by haworku** [comment]
Got a new link and everything is working now!

File: services/app-web/src/pages/App/App.tsx [hunk 1]
```diff
@@ -35,9 +36,14 @@ function App({
...
 }): React.ReactElement {
-    useEffect(() => {
-        logEvent('on_load', { success: true })
-    }, [])
+
+    // This is a hacky way to fake feature flags before we have feature flags.
+    // please avoid reading env vars outside of index.tsx in general.
+    const environmentName = process.env.REACT_APP_STAGE_NAME || ''
+    const isProdEnvironment = ['prod', 'val'].includes(environmentName)
+    const jiraTicketCollectorURL = `https://meghantest.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/-9zew5j/b/7/c95134bc67d3a521bb3f4331beb9b804/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector.js?locale=en-US&collectorId=e59b8faf`
```
Comment (macrael) [inline services/app-web/src/pages/App/App.tsx:44]: Is this a secret? I guess it just ends up in our app so there’s no way to hide it really.

File: services/app-web/src/pages/App/App.tsx [hunk 2]
```diff
@@ -35,9 +36,14 @@ function App({
...
+    
+    useScript(jiraTicketCollectorURL, !isProdEnvironment)
```
Comment (macrael) [inline services/app-web/src/pages/App/App.tsx:46]: Could we wrap the useScript call inside `if (!isProdEnvironment)` or does it need to be called every time?

File: services/app-web/src/pages/App/App.tsx [hunk 3]
```diff
@@ -35,9 +36,14 @@ function App({
     apolloClient: ApolloClient<NormalizedCacheObject>
     s3Client: S3ClientT
 }): React.ReactElement {
-    useEffect(() => {
-        logEvent('on_load', { success: true })
```
Comment (macrael) [inline services/app-web/src/pages/App/App.tsx:39]: Can you check with Mojo to make sure we don’t need this anymore? I’m not sure if we are logging an event on load through otel yet.

**Review by macrael — APPROVED** [review]
Yay, nice and easy.

File: services/app-web/src/pages/App/App.tsx (hunk 1 above)
Comment (haworku) [reply services/app-web/src/pages/App/App.tsx:44]: I have no idea

File: services/app-web/src/pages/App/App.tsx (hunk 2 above)
Comment (haworku) [reply services/app-web/src/pages/App/App.tsx:46]: According to [rule of hooks](https://reactjs.org/docs/hooks-rules.html#explanation) can't conditionally run a hook

File: services/app-web/src/pages/App/App.tsx (hunk 3 above)
Comment (haworku) [reply services/app-web/src/pages/App/App.tsx:39]: Let me just add that line back, I think we can clear out logEvent later I was getting hasty

**Comment by haworku** [comment]
I will need to open a new branch to get this work through. getting api fails similar to our bug earlier last week. think my AWS env on this review app is not set up properly

**Pull request closed without merging**